main/src/main/java/com/sedmelluq/discord/lavaplayer/player/DefaultAudioPlayerManager.java

```diff
@@ -194,7 +177,10 @@ public Future<Void> loadItem(final AudioReference reference, final AudioLoadResu
   @Override
   public Future<Void> loadItemOrdered(Object orderingKey, final AudioReference reference, final AudioLoadResultHandler resultHandler) {
     try {
-      return orderedInfoExecutor.submit(orderingKey, createItemLoader(reference, resultHandler));
+      return orderedInfoExecutor.submit(orderingKey, () -> {
+        loadItem(reference, resultHandler);
```

Comment:
Shouldn't this be `loadItemSync`? Looks like a stack overflow otherwise

Reply:
oh yeah, lol